### DISCUSSION TITLE
Typo in Usage Section of Documentation

### DISCUSSION THREAD

**Discussion by jayrajput** [ahyatt/ekg#37] [Q&A]
Should it be `n` and `p` and not `f` and `b` in the usage section

> The basic read operation in ekg is to show a list of notes according to some rule. An example is ekg-show-notes-with-tag. The notes displayed can be navigated between using f and b, and interacted with. The following section has a complete list of commands and keybindings.

**Comment by ahyatt** [comment]
Thanks for noticing that mistake.  I've pushed a fix to both the `main` and `develop` branch.
